src/components/dictionaryConcepts/components/helperFunction.js

```diff
@@ -52,6 +52,8 @@ export const DATA_TYPES = [
 export const MAP_TYPES_DEFAULTS = ['SAME-AS', 'NARROWER-THAN'];
 
 export const INTERNAL_MAPPING_DEFAULT_SOURCE = 'CIEL';
+// todo This seems like we are hard coding CIEL as the only external source, which is unlikely
```

Comment:
What are you using these comments for?

Reply:
These can be really helpful for finding areas where things were hard coded as we get into the next phase. Do you think they are not as important?

Reply:
Shouldn't you instead create tickets for such?

Reply:
Alright. I'll note these down separately and create tickets.